src/cli/rustup_mode.rs

```diff
@@ -135,536 +128,567 @@ pub fn main() -> Result<utils::ExitCode> {
     cfg.check_metadata_version()?;
 
     Ok(match matches.subcommand() {
```

Comment:
This introduces a lot of noise.

What about

```
Ok(match matches.subcommand() {
  OK(s) => match s {
    ("dump-testament", _) => common::dump_testament()?,
     ...
     },
   _ => unreachable()
   }
```

Reply:
How about now?

Reply:
How about now?